integration-libs/punchout/styles/_punchout-template.scss

```diff
@@ -41,6 +41,7 @@
       min-width: 0;
     }
   }
+  min-height: calc(100vh - 1px);
```

Comment:
why this value?
is it standard in spartacus?

Reply:
Here (in eg, dark theme) we have some weird 1px bottom border of `header` element. So that's why applied `- 1px`